File: SeleniumTestToolQA/pages/ToolsQAHomePage.java
package SeleniumTestToolQA.pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class ToolsQAHomePage {

    WebDriver driver;
    By formsLink = By.xpath("//h5[text()='Forms']");

    public ToolsQAHomePage(WebDriver driver){
        this.driver=driver;
    }

    public void goTo() {
        driver.get("https://demoqa.com/");
    }

    public ToolsQAFormsPage openForms() {
        driver.findElement(formsLink).click();
        return new ToolsQAFormsPage(driver);
    }

}

File: PlaywrightToolsQA/ToolsQAHomePage.js
const { expect} = require('@playwright/test');

exports.ToolsQAHomePage = class ToolsQAHomePage {

    /**
     * @param page {import('playwright/test').Page} page
     */
    constructor(page) {
        this.page = page;
        this.formsLink = page.locator("text=Forms");
    }

    async goto() {
        await this.page.goto("https://demoqa.com/");
    }

    async openForms() {
        await this.formsLink.first().click();
    }
}

